kvm.go

```diff
@@ -627,6 +627,7 @@ func (d *Driver) getIPByMacFromSettings(mac string) (string, error) {
 	type Lease struct {
 		Ip_address  string `json:"ip-address"`
 		Mac_address string `json:"mac-address"`
+		Expiry_time string `json:"expiry-time"`
```

Comment:
Oops I must have missed this. I was the one who patched the KVM driver in minikube :). I will remove